scripts/start.sh

```diff
@@ -64,7 +64,7 @@ if [ "$architecture" == "arm64" ]; then
             ;;
     esac
     
-    sed -i "s|\(\"\$UE_PROJECT_ROOT\/Pal\/Binaries\/Linux\/PalServer-Linux-Test\" Pal \"\$@\"\)|LD_LIBRARY_PATH=/home/steam/steamcmd/linux64:\$LD_LIBRARY_PATH $box64_binary \1|" ./PalServer-arm64.sh
+    sed -i "s|\(\"\$UE_PROJECT_ROOT\/Pal\/Binaries\/Linux\/PalServer-Linux-Shipping\" Pal \"\$@\"\)|LD_LIBRARY_PATH=/home/steam/steamcmd/linux64:\$LD_LIBRARY_PATH $box64_binary \1|" ./PalServer-arm64.sh
```

Comment:
Do we want to have an if statement to support older versions?

Reply:
No I don't think so, people wanting to stay on a specific version of the game, would have to pin the container version. 
Backwards compatibility is very hard to support in an early access game .